tsl/src/bgw_policy/job.c

```diff
@@ -89,14 +89,65 @@ get_chunk_id_to_reorder(int32 job_id, Hypertable *ht)
 															 -1);
 }
 
+static int64
+get_compression_window_end_value(Dimension *dim, Jsonb *config)
```

Comment:
If I noticed correctly, there are three different names are used for identifiers in the code for the same value: `older_than`, `lag`, `window_end`. If it is so, it is confusing and would be nice to use only one name.

Reply:
older_than and lag are used interchangeably, i started using lag because i prefer it over older_than but what we name this should by in sync with external api where we havent decided on the new name yet.
window_end is not equal to those `window_end = now() - lag`

Reply:
Thank you for clarifying about `window_end`. I was confused, since arguments to `ts_interval_from_now_func_get_datum` didn't fit the window during review on GH website.